configs/dyhead/README.md

```diff
@@ -0,0 +1,61 @@
+# Dynamic Head: Unifying Object Detection Heads with Attentions
+
+## Abstract
+
+<!-- [ABSTRACT] -->
```

Comment:
The format of readme is updated. See changes in https://github.com/open-mmlab/mmdetection/pull/7051. Could you apply the newest change to this file?

Reply:
pre-commit does not pass before merging 7051.